src/main/webapp/app/quiz/participate/quiz.component.html

```diff
@@ -19,6 +19,19 @@ <h2 [ngSwitch]="mode">
     <div class="container">
         <div class="quiz-footer-content">
             <div>
+                <div [hidden]=" quizExercise.questions == null">
+                    <div class="stepwizardquiz">
```

Comment:
I think this and the previous `<div>` can be merged in just once, since this is the only child.

Something like
`<div [hidden]=" quizExercise.questions == null" class="stepwizardquiz">`

It helps keeping the DOM clean :-)

Reply:
deleted